core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -119,14 +119,9 @@ find_numbers(<<"+", Rest/binary>>, Quantity, Options) ->
 find_numbers(<<"1", Rest/binary>>, Quantity, Options) ->
     find_numbers(Rest, Quantity, Options);
 
-find_numbers(<<Prefix:3/binary, _/binary>>=Num, Quantity, Options) when ?IS_US_TOLLFREE(Prefix) ->
-    <<"8", Second:1/binary, _/binary>> = Prefix,
-    ToSearch = case knm_converters:is_reconcilable(Num) of
-                   'true' -> Num;
-                   'false' -> Second
-               end,
-
-    Params = [ "tollFreeWildCardPattern=8", binary_to_list(ToSearch), "*"
+find_numbers(<<Prefix:3/binary, _/binary>>=_Num, Quantity, Options) when ?IS_US_TOLLFREE(Prefix) ->
+    <<_:1/binary, Wildcard/binary>> = Prefix,
```

Comment:
`Prefix` doesn't look used outside of this. Can you move this matching to the line above @lazedo ?